.env

```diff
@@ -0,0 +1 @@
+DB_HOST = mongodb+srv://arkdau:Julia2013r@cluster0.lf3mcwn.mongodb.net/db-contacts?retryWrites=true&loadBalanced=false&replicaSet=atlas-odqf83-shard-0&readPreference=primary&srvServiceName=mongodb&connectTimeoutMS=10000&authSource=admin&authMechanism=SCRAM-SHA-1
```

Comment:
Tego pliku nie wysyłaj ponieważ to jest prywatna informacja i nie powinieneś się nią dzielić na repozytorium. 
Stwórz sobie .env.example gdzie umieścisz
`DB_HOST=` 
I tylko tyle. To będzie informacja dla mnie i dla ciebie w przyszłości że tu trzeba będzie wpisać link do bazy danych.